pkgs/development/libraries/clfft/default.nix

```diff
@@ -1,21 +1,33 @@
-{ lib, stdenv, fetchFromGitHub, cmake, fftw, fftwFloat, boost166, opencl-clhpp, ocl-icd }:
+{ lib, stdenv, fetchFromGitHub, cmake, fftw, fftwFloat, boost, opencl-clhpp, ocl-icd, darwin }:
 
+let
+  inherit (darwin.apple_sdk.frameworks) OpenCL;
+in
 stdenv.mkDerivation rec {
   pname = "clfft";
   version = "2.12.2";
 
   src = fetchFromGitHub {
     owner = "clMathLibraries";
     repo = "clFFT";
-    rev = "refs/tags/v${version}";
-    sha256 = "134vb6214hn00qy84m4djg4hqs6hw19gkp8d0wlq8gb9m3mfx7na";
+    rev = "v${version}";
+    hash = "sha256-yp7u6qhpPYQpBw3d+VLg0GgMyZONVII8BsBCEoRZm4w=";
   };
 
   sourceRoot = "source/src";
 
+  postPatch = ''
+    sed -i '/-m64/d;/-m32/d' CMakeLists.txt
```

Comment:
My only worry is this is too broad and could cause quiet weirdness in future versions of `CMakeLists.txt`